### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by cbeer**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -7,12 +7,11 @@ gem 'rake', '~> 10.3'
...
-gem 'lyber-core', '~> 3.2', '>=3.2.4'
+gem 'lyber-core', '~> 4.0'
 gem 'robot-controller', '~> 1.0' # requires Resque
 
 group :development, :test do
   gem 'rspec'
   gem 'yard'
   gem 'dlss-capistrano'
-  gem 'holepicker', '~> 0.3', '>= 0.3.3'
 end
```
Comment (darrenleeweber) [inline Gemfile]: Could this group be split up?  A 'deployment' group could split out the capistrano dependencies so they don't creep into development deployments.  rspec isn't required for a development deployment.

**Comment by darrenleeweber** [comment]
Generally :+1: with this.

File: Gemfile (hunk 1 above)
Comment (cbeer) [inline Gemfile]: Sure. As-is, the whole deployment for this project seems out-of-sync with downstream practice.

**Comment by darrenleeweber** [comment]
I suggest @drh-stanford should take a look at this to merge it.

**Pull request merged**